tests/baselines/reference/typeParameterFixingWithContextSensitiveArguments3.errors.txt

```diff
@@ -11,6 +10,6 @@ tests/cases/compiler/typeParameterFixingWithContextSensitiveArguments3.ts(7,35):
     
     var d = f(a, b, u2 => u2.b, t2 => t2);
                                       ~~
-!!! error TS2322: Type 'A' is not assignable to type 'B'.
-!!! error TS2322:   Property 'b' is missing in type 'A'.
+!!! error TS2737: Property 'b' is missing in type 'A' but present in type 'B'.

```

Comment:
Part of me feels like we lose some context on why we're reporting the error message when it's the first elaboration.